lib/prawn/markup/processor/tables.rb

```diff
@@ -86,6 +86,11 @@ def inside_container?
       def add_cell_text_node(cell, options = {})
         return unless buffered_text?
 
+        options[:font_style] ||= options.delete(:style) if options.key?(:style)
+        # only allow on supported options of prawn-table; See https://github.com/prawnpdf/prawn-table/blob/master/manual/table/cell_text.rb
+        options.slice!(*%i[font font_style inline_format kerning leading min_font_size
+                           overflow rotate rotate_around single_line size text_color valign])
```

Comment:
The options hash must not be modified in this method. Use `options = options.slice(...)` to create a new object. Furthermore, `slice!` is not in the ruby stdlib anyways.